### PR TITLE
Inkgd docs: Add Sphinx initial files.
These files are necessary to setup a document in Sphinx.
This will be later used in ReadTheDocs: https://inkgd.readthedocs.io/

### PR DESCRIPTION
**Pull request by videlanicolas**

### CODE REVIEW

**Comment by videlanicolas** [comment]
#41

**Review by ephread — COMMENTED** [review]
LGTM!

I just have a question regarding `make.bat`, as someone who has never used Sphinx before. Isn't that file windows-only? How does the whole system behave on Linux and macOS?

File: docs/source/index.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Inkgd: Ink for Godot – *master* branch
```
[the remark this answers is not in the document]
Comment (videlanicolas) [reply docs/source/index.rst]: It's not automatically generated, I copy/pasted an example that expects a *master* branch. Given Github moved away from master for main instead, I've changed this in this file.

**Comment by videlanicolas** [comment]
> Isn't that file windows-only?

This was automatically generated by Sphinx, but my understanding is that the command `make html` should work out of the box on any OS. `make.bat` will run on Windows systems and `makefile` will run on Linux and macOS.

**Comment by ephread** [comment]
> `make.bat` will run on Windows systems and `makefile` will run on Linux and macOS.

Makes total sense, I hadn't realised `makefile` and `make.bat` served the same purpose. 🤦
Thanks for the explanation, merging now!

**Review by ephread — APPROVED** [review]

**Pull request merged**